pybossa/view/projects.py

```diff
@@ -449,13 +449,14 @@ def clone_project(project, form):
 
     proj_dict['owner_id'] = current_user.id
     proj_dict['owners_ids'] = [current_user.id]
-    proj_dict['short_name'] = form['short_name']
     proj_dict['name'] = form['name']
-    replacement = 'pybossa.run("%s")' % proj_dict['short_name']
-    pybossa_re = "(pybossa\.run\(\s*)(['\"]\S+['\"]\))"
-    proj_dict['info']['task_presenter'] = re.sub(pybossa_re,
-                                                 replacement,
-                                                 proj_dict['info'].get('task_presenter', ''))
+
+    task_presenter = proj_dict['info'].get('task_presenter', '')
+    regex = r"([\"\'\/])({})([\"\'\/])".format(proj_dict['short_name'])
```

Comment:
This looks good and works well. Just to make the code a little more clear, one item to consider is naming the variables, and perhaps a comment to explain the regex? :)

```python
old_name = proj_dict['short_name']
new_name = form['short_name']

# Replace all instances of old_name in the task presenter HTML with new_name.
regex = r"([\"\'\/])({})([\"\'\/])".format(old_name)
proj_dict['info']['task_presenter'] = re.sub(regex, r"\1{}\3".format(new_name), task_presenter)